src/pages/Account/AccountSecurity.tsx

```diff
@@ -1,10 +1,68 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Alert } from 'reactstrap';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { User } from 'firebase';
 
-class AccountSecurity extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Security Page</h1>
-    );
+import { getDisplaySuccessMessage, SuccessMessage, getDisplayErrorMessage, ErrorMessage } from 'utils/validators';
+import { resetPassword } from 'utils/firebase';
+import { AlertProperties } from 'components/work/Alert/Alert';
+
+function AccountSecurity() {
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '', show: false });
+  const [isSubmitting, setSubmitting] = React.useState<boolean>(false);
+
+  return (
+    <>
+      <Alert
+        isOpen={alert.show}
+        color={alert.color}
+        toggle={() => setAlert({ ...alert, show: false })}>
+        {alert.msg}
+      </Alert>
+
+      <FirebaseConsumer>
+        {({ user }: FirebaseUserProps) => (
+          <ListGroup className="mb-2 d-flex flex-column">
+            {user &&
+              <ListGroupItem
+                className="w-100 d-flex flex-column"
+                disabled={isSubmitting}
+                onClick={() => handleResetPasswordClick(user)}>
+                <h6 className="mb-0">Reset Password</h6>
+                <div className="d-flex justify-content-between">
+                  <h6 className="mb-0 small text-muted">Click here to reset your password</h6>
+                  <i className="fas fa-lock"></i>
+                </div>
+              </ListGroupItem>
+            }
+          </ListGroup>
+        )}
+      </FirebaseConsumer>
+    </>
+  );
+
+  function handleResetPasswordClick(user: User) {
+    if (!user.email) return;
```

Comment:
do we need some sort of error/redirect if the user's email is missing?

Reply:
this line was mainly to satisfy typescript. Should we redirect to login page if user or user.email does not exist?

Reply:
prompting an error would work

Reply:
i think if we ran into a case where the user.email doesn't exist maybe we should prompt an error and bring them to the login screen?

Reply:
put in a reactstrap alert for now.